test/system/010-images.bats

```diff
@@ -344,7 +344,7 @@ Deleted: $pauseID"
 driver="overlay"
 
 [storage.options]
-additionalimagestores = [ "$imstore/root" ]
+additionalimagestores = [ "$imstore/root|NO_REUSE" ]
```

Comment:
We can invent an infinite number of new options. Please don’t add ambiguous in-line syntaxes unless there is really no other alternative.

Reply:
I agree, this is just merely a compact version to collect feedback. There should be more agreeable manners to achieve it.